### PR TITLE
dataclients/kubernetes: log duplicate route error only when it differs

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
This should reduce log volume

For #2394

### CODE REVIEW

File: dataclients/kubernetes/kube.go [hunk 1]
```diff
@@ -383,19 +383,17 @@ func ParsePathMode(s string) (PathMode, error) {
 func mapRoutes(r []*eskip.Route) (map[string]*eskip.Route, []*eskip.Route) {
 	var uniqueRoutes []*eskip.Route
 	m := make(map[string]*eskip.Route)
-	for _, ri := range r {
-		if existingRoute, ok := m[ri.Id]; ok {
-			if eskip.Eq(existingRoute, ri) {
-				log.Warnf("Duplicate routeID found: %s", ri.Id)
-			} else {
-				log.Errorf("Failed to map route, duplicate ID with different body found, existing routeID: %s, existing route body: %s, new route body: %s", ri.Id, existingRoute.String(), ri.String())
+	for _, route := range r {
```
Comment (szuecs) [inline dataclients/kubernetes/kube.go:386]: Maybe for _, r := range routes ?

Comment (AlexanderYastrebov) [reply dataclients/kubernetes/kube.go]: Renamed variables

**Comment by MustafaSaber** [comment]
👍

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**